test/tests/CookieBanner-test.js

```diff
@@ -111,4 +111,33 @@ describe('CookieBanner', () => {
     expect(_myComponent.length).toBe(1, 'cookie banner is not displaing custom child component');
   });
 
+  it('should be replaced with custom child component using function', () => {
+
+    const MyOtherComponent = React.createClass({
+      render() {
+        return <div className='my-other-component' onClick={this.props.onAccept}/>;
+      }
+    });
+
+    MyOtherComponent.propTypes = {
```

Comment:
`PropTypes` don't add much value to the test here. I would just remove this part to keep the code shorter.

(in general - by the way - we use @props from `tcomb-react` instead of `PropTypes`)